src/components/endpointCreation/NewCaptureModal.tsx

```diff
@@ -84,17 +84,20 @@ function NewCaptureModal(
 
     const [searchParams, setSearchParams] = useSearchParams();
     const sourceTypeParam = searchParams.get('sourcetype');
-    const { isFetching, schema, error, image } =
-        useSourceSchema(sourceTypeParam);
+    const { isFetching, schema, error, image } = useSourceSchema(
+        sourceTypeParam ? sourceTypeParam : ''
+    );
 
     const [newCaptureDetailsFormData, setNewCaptureDetailsFormData] = useState<{
         tenantName: string;
         captureName: string;
         sourceType: string;
+        sourceImage: string;
```

Comment:
We want to start passing around the full image path as that is what is really being used.